src/scss/constants.scss

```
@@ -106,6 +110,16 @@ $z-layers: (
   venuepage-preview-indication: 1
 );
 
+@font-face {
+  font-family: "speakTweak";
+  src: url("../assets/fonts/speakTweak.ttf");
+}
+
+@font-face {
+  font-family: "zoinks";
+  src: url("../assets/fonts/zoinks.ttf");
+}
```

Will this work to load from the absolute path?

```suggestion
@font-face {
  font-family: "speakTweak";
  src: url("assets/fonts/speakTweak.ttf");
}

@font-face {
  font-family: "zoinks";
  src: url("assets/fonts/zoinks.ttf");
}
```

---

fixed in https://github.com/sparkletown/sparkle/pull/1240/commits/4b4351f2e7c0d9b970954dc3303bfa8db9db3a37